packages/fast-usdc/test/cli/operator-commands.test.ts

```diff
@@ -0,0 +1,65 @@
+import test from 'ava';
+import { Command } from 'commander';
+import { addOperatorCommands } from '../../src/cli/operator-commands.js';
+import { MockCctpTxEvidences } from '../fixtures.js';
+import { makeMarshal } from '@endo/marshal';
+import type { Writable } from 'node:stream';
+
+export const flags = (
+  record: Record<string, string | number | bigint | undefined>,
+): string[] => {
+  // @ts-expect-error undefined is filtered out
+  const skipUndef: [string, string][] = Object.entries(record).filter(
+    ([_k, v]) => v !== undefined,
+  );
+  return skipUndef.map(([k, v]) => [`--${k}`, `${v}`]).flat();
+};
+
+const marshalData = makeMarshal(_v => assert.fail('data only'));
+
+const mockStream = <T extends Writable>(buf: string[]): T =>
```

Comment:
Could you please put this in `testing/mocks`? It's better than the existing `mockOut` thing so should be used going forward.

Reply:
We don't export `testing` from packages. Mocks are often useful outside a package so consider `fast-usdc/tools/testingMocks.js` or somesuch.